docs/markdown/Writing Plugins/common-plugin-tasks/plugins-general-subsystems.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Maybe add this documentation page in a separate PR in order to keep this PR on one topic?

Reply:
@benjyw: thoughts on above question?

Reply:
I'm neutral on that question, since the docs are pertinent to the code change, even if wider.